app/resources/forms.py

```diff
@@ -15,18 +18,22 @@ class ResourceForm(Form):
         InputRequired(),
     ])
     website = StringField('Website')
-    street_address = StringField('Street Address', validators=[
+
+    street_number = IntegerField('Street Number', validators=[
         InputRequired()
     ])
-    city = StringField('City', validators=[
+    route = StringField('Street Address', validators=[
```

Comment:
Is this what you meant by having to change form field names to match the schema of Google Maps? If not, why is it 'route' and not 'Street Address'?